src/tools/compiletest/src/common.rs

```diff
@@ -151,7 +145,7 @@ pub enum Sanitizer {
 ///
 /// FIXME: audit these options to make sure we are not hashing less than necessary for build stamp
 /// (for changed test detection).
-#[derive(Debug, Default, Clone)]
+#[derive(Debug, Clone)]
```

Comment:
Remark: I was wondering what a default `Config` even means, turns out it's simply unused. Ditto on test mode.